Corona-Energy-Grid-Frontend/src/app/localdata/shared-data.service.ts

```diff
@@ -6,12 +6,12 @@ import { BehaviorSubject } from 'rxjs';
 })
 export class SharedDataService {
 
-  private authSource = new BehaviorSubject(false);
+  private authSource = new BehaviorSubject('');
```

Comment:
wordt deze nog ergens in de code gebruikt? anders kunnen we deze weghalen, en de verantwoordelijkheid bij de auth service neerleggen, aangezien die over het authenticatieproces gaat

Reply:
Deze kan weg, wordt inderdaad niet gebruikt.